pinot-common/src/main/java/org/apache/pinot/common/utils/helix/TableCache.java

```diff
@@ -98,7 +102,7 @@ public synchronized void refresh() {
             try {
               TableConfig tableConfig = TableConfigUtils.fromZNRecord(znRecord);
               String tableNameWithType = tableConfig.getTableName();
-              _tableConfigMap.put(tableNameWithType, tableConfig);
+              _tableConfigMap.put(tableNameWithType.toLowerCase(), tableConfig);
```

Comment:
Not sure why we lower case here. Can you elaborate?

Reply:
Reverted. I was following the pattern with column and table name in this class.